pymatgen/entries/computed_entries.py

```diff
@@ -28,7 +27,7 @@
 __date__ = "Apr 30, 2012"
 
 
-class ComputedEntry(MSONable):
+class ComputedEntry(Entry):
     """
     An lightweight ComputedEntry object containing key computed data
     for many purposes. Extends a PDEntry so that it can be used for phase
```

Comment:
@ayushsgupta let's change the docstring to:

'Lightweight Entry object for computed data. Contains facilities for applying corrections to the .energy attribute and for storing calculation parameters`